### ISSUE TITLE
Could you add support for VideoRaylib?

### ISSUE THREAD

**Issue by demid-code** [anrayliu/pyvidplayer2#50]
I saw that you have many bindings already, like pyglet, pygame, PyQT, PySide. Could you add suport for raylib too?

I am working on video editor in python with pyray (python raylib binding). I tested your library in pygame and it works pretty cool, but i prefer to work with raylib and want to use your library :)

**Comment by anrayliu** [comment]
Yes, I can definitely take a look at pyray and see what I can do. Although, you'll need to give me some time because I am currently working on a rather large stability update. If everything goes well, pyray support and the bug fixes will come out in version 0.9.25. I'll let you know if I have any updates. Cheers.

**Comment by anrayliu** [comment]
Support for raylib has now been added with v0.9.25. Check out this [example](https://github.com/anrayliu/pyvidplayer2/blob/main/examples/raylib_demo.py) to see how it works.
